docs/xxcodecachetotalmaxrampercentage.md

```diff
@@ -0,0 +1,52 @@
+<!--
+* Copyright (c) 2017, 2022 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0 OR GPL-2.0-only WITH OpenJDK-assembly-exception-1.0
+-->
+
+# -XX:codecachetotalMaxRAMPercentage
+
+This option sets a maximum limit for the total memory that the VM can allocate for code caches.
+
+## Syntax
+
+        -XX:codecachetotalMaxRAMPercentage=<number>
+
+| Setting                 | Value      | Default                     |
+|-------------------------|------------|:---------------------------:|
+| `<number>`    | Between 1 and 100 (fractional numbers are allowed) |  25                       |
+
+Where `<number>` is the maximum limit expressed as a percentage of the available physical memory.
+
+For example, if you specify `-XX:codecachetotalMaxRAMPercentage=30`, the VM is not allowed to use more than 30% of the available physical memory for the internal code caches.
+
+## Explanation
+
+In environments with low physical memory availability, the VM might use too much memory for code caches, leaving little memory for the VM or JIT data structures. With the `-XX:codecachetotalMaxRAMPercentage` option, you can reduce the default total code cache value (the default is set to 256 MB for 64-bit systems and 64 MB for 31 or 32-bit systems). In environments with plenty of memory, the `-XX:codecachetotalMaxRAMPercentage` setting is ignored.
```

Comment:
How about this:

By default, the total size of the JIT code cache is 256 MB for a 64-bit VM and 64 MB for a 31/32-bit VM.
Despite this limit, in environments with low physical memory availability, the VM might use too much memory for code caches, leaving little memory for the VM or JIT data structures. With the `-XX:codecachetotalMaxRAMPercentage` option, you can reduce (but not increase) the default total size of the JIT code cache. 
If you want to increase the total code cache size, you can use the [`-XX:codecachetotal`](xxcodecachetotal.md) or the [`-Xcodecachetotal`](xcodecachetotal.md) options.

Reply:
> How about this:
> 
> By default, the total size of the JIT code cache is 256 MB for a 64-bit VM and 64 MB for a 31/32-bit VM. Despite this limit, in environments with low physical memory availability, the VM might use too much memory for code caches, leaving little memory for the VM or JIT data structures. With the `-XX:codecachetotalMaxRAMPercentage` option, you can reduce (but not increase) the default total size of the JIT code cache. If you want to increase the total code cache size, you can use the [`-XX:codecachetotal`](xxcodecachetotal.md) or the [`-Xcodecachetotal`](xcodecachetotal.md) options.

@mpirvu - One last question - :) In this you have said that "With the `-XX:codecachetotalMaxRAMPercentage` option, you can reduce (**but not increase**) the default total size of the JIT code cache." But in this  https://github.com/eclipse-openj9/openj9-docs/pull/1114#issuecomment-1609990807, you have mentioned that "If the user finds that 25% is too much or too little, he can tweak that percentage value to something appropriate for his system." and that "If the application uses little Java heap but compiles many methods and needs a larger code cache, then the user might want to use a percentage value larger than 25%." So doesn't that mean that we can actually increase the default total size of the JIT code cache.?